src/types/cart.d.ts

```diff
@@ -200,7 +200,8 @@ export interface CartEndpoint
     quantity?: number,
     data?: any,
     isSku?: boolean,
-    token?: string
+    token?: string,
+    additionalHeaders?: any
```

Comment:
hey @rishi-ayata could you be a little more specific with the type? this should always be an object